src/core/base.jl

```diff
@@ -77,41 +96,64 @@ function InitializeInfrastructureModel(InfrastructureModel::Type, data::Dict{Str
 end
 
 
+function ref_initialize(data::Dict{String,<:Any}, global_keys::Set{String} = Set{String}())
```

Comment:
I think it makes sense to keep these as separate functions mostly because single infrastructure models will require specifying a default `it` argument when calling `ref_initialize`, but the multi-infrastructure piece of `ref` does not require this. Let's discuss more on Thursday.

Reply:
We will have two `ref_initialize` functions: one which takes in a specific infrastructure type and returns the proper data object and one that does not take in a specific infrastructure type and only works with multi-infrastructure data. The first function will be used within packages related to a specific infrastructure.